testing/mock/ibc_module.go

```diff
@@ -154,6 +154,15 @@ func (im IBCModule) OnTimeoutPacket(ctx sdk.Context, packet channeltypes.Packet,
 	return nil
 }
 
+// SendPacket implements the ICS4 Wrapper interface
+func (im IBCModule) SendPacket(
```

Comment:
Added this in an attempt to make the mock ibc module adhere to the middleware interface. Didn't seem to work. 

```
testing/simapp/app.go:408:21: cannot use controller.NewIBCMiddleware(app.ICAControllerKeeper, icaControllerStack) (type controller.IBCMiddleware) as type mock.IBCModule in assignment
```